plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -72,6 +72,44 @@ type Resource struct {
 	OpaqueIntResources map[v1.ResourceName]int64
 }
 
+func NewResource(rl v1.ResourceList) *Resource {
```

Comment:
.... This is only used for unit tests ?  if so comment as such or move into the `_test.go` file ?